spec/directives/forward/with.hrx

```diff
@@ -656,3 +656,65 @@ b {
   midstream1: overridden 1;
   midstream2: overridden 2;
 }
+
+<===>
+================================================================================
+<===> use/multiple/same_file_works/input.scss
+@use 'module' with (
+  $list-variable: 'foo',
+  // $other: 'baz', <- uncommenting this is OK
+  // $whatever: 'bar', <- uncommenting this causes the module loaded twice err
```

Comment:
Because of the behavior with these two
I am thinking that when a variable exists in all `@forwarded` declarations then it's fine, but when there is a variable that does not exist somehow it triggers the "loaded twice" error

So whatever is dedupping the module seems to have a quirk with variables

or something like that :)